writeup/report/final_report.tex

```diff
@@ -51,40 +51,41 @@
 \maketitle
 
 \begin{abstract}
-    Many problems require a well defined notion of a distance between points in space.
+    Many learning problems require a well defined notion of distance between points in space. 
     Constructing or finding such a measure falls into the field of metric learning.
-    Although many algorithms exist in the field when a learner has access to a fixed dataset,  there is room for improvement in terms of samples efficiency that the learner needs to know, imposition of desired structure, especially when the data appears in an \textit{online} manner.
-    We propose a project that reduces the problem of online/active metric learning to bandits. In case our plan turn out to be too ambitious, we have a fallback - an empirical investigation of some algorithms that have dealt with the problem in an online setting or in situations where the learner can selective query the points that it wants to know information about.
+    Applications where data is collected in an interactive or online fashion have become commonplace. 
+    A lot of downstream inference assumes an appropriate representation of this incoming data.
+    To contribute to this problem, we attempt to formulate similarity learning in a bandits framework. 
+    We exploit LinUCB like methods, in both online as well as active settings to design and empirically investigate algorithms 
+    that try to learn with linear as well as more expressive non-linear similarity function classes.
 \end{abstract}
 
 
 \section{Introduction}
+Contextual bandits are a framework that leverage the ability to act and potentially influence the incoming data distribution. In a traditional contextual bandits problem in each round t, the learner observes some context c, chooses some action a to perform, and receives some reward r(c,a) based on the context and action. The learners goal is to maximize its total rewards over a fixed number of total rounds. A lot of interactive repeated decision problems can be modelled using contextual bandits. They serve as a simplified version of the reinforcement learning framework, while providing a toolset needed to analyse performance of the algorithm. This estimate is usually measured by regret, an expected difference between the sum of optimal rewards and collected rewards. Online learning algorithms are designed with the informal goal of minimizing prediction errors. These goals, if aligned can allow the construction of bandit algorithms that solve online learning problems. \\
```

Comment:
Could you put the relevant terms like c, a and r(c,a) in the math symbols (like $r(c,a)$)